src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -6,3 +6,96 @@
 from azure.cli.help_files import helps #pylint: disable=unused-import
 
 #pylint: disable=line-too-long
+
+helps['network dns record-set create'] = """
```

Comment:
I'm fine with that, but I think Johan would argue that we incur more "help debt" by writing our own custom help instead of pushing the SDK to fix their help. 

---
In reply to: [75560645](https://github.com/Azure/azure-cli/pull/718#discussion_r75560645) [](ancestors = 75560645,75556265)